templates/_helpers.tpl

```diff
@@ -22,3 +22,41 @@ We truncate at 63 chars because some Kubernetes name fields are limited to this
 {{- end -}}
 {{- end -}}
 {{- end -}}
+
+{{/*
+Create a livenessProbe.
+Allow the default value to be completely overriden by an optional value.
+Retain he original livenessProbe logic.
+*/}}
+{{- define "livenessProbe" -}}
```

Comment:
For consistency with other chart-specific definitions, include a `docker-registry.` prefix here and on the `readinessProbe` definition.

Reply:
Added prefix.